src/Binary/Loader/ChainLoader.php

```diff
@@ -37,36 +39,14 @@ public function find($path)
     {
         $exceptions = [];
 
-        foreach ($this->loaders as $loader) {
+        foreach ($this->loaders as $configName => $objectInst) {
```

Comment:
```suggestion
        foreach ($this->loaders as $configName => $loader) {
```
Just a suggestion that keeping `$loader` as variable seems more understandable to me.

Reply:
Sounds good; done.

Reply:
Sounds good; done.

*(Note: I do agree that the use of `loader` (especially for user-facing code) is much clearer using the suggested (and original) variable name (instead of `objectInst`); in fact, I never actually intended to change that variable at all! ;-) I think the loader variable was altered when I was refactoring the new exception class properties and accidentally cast too wide a "find/replace" net, such that the ChainLoader::$loader property was mistakenly included in the change. Regardless, it's fixed now. I was working on a machine without my normal environment, PHPStorm, and was instead using SublimeText, so such unexpected changes shouldn't normally be an issue. Thanks for the catch, though.)*